src/ochanticipy/datasources/codab/codab.py

```diff
@@ -30,12 +31,30 @@ def __init__(self, country_config: CountryConfig):
             is_public=True,
             config_datasource_name="codab",
         )
-        self._raw_filepath = (
-            self._raw_base_dir / f"{self._country_config.iso3}_"
-            f"{self._datasource_base_dir}.shp.zip"
+        self._base_dir = Path(
+            f"{self._country_config.iso3}_{self._datasource_base_dir}"
         )
 
-    def download(self, clobber: bool = False) -> Path:
+        # account for files stored across multiple resources
+        res_name = self._datasource_config.hdx_resource_name
+        self._multiple_resources = isinstance(res_name, list)
+
+        if self._multiple_resources:
+            self._resource_dirs = [
```

Comment:
You don't need to make this an instance variable if it's not used outside of the method

Reply:
It's used in the `_load_admin_layer()`